middleware.go

```diff
@@ -125,3 +127,43 @@ func deleteFields(content map[string]interface{}) map[string]interface{} {
 
 	return content
 }
+
+func shouldNotify(r *http.Request) bool {
+	// is not exception url
+	if !ArrayContains(ExceptionURLS, r.RequestURI) {
+		return true
+	}
+
+	key := getRateLimitKey(getIPAddress(r), r.RequestURI)
+
+	timestamp, found := notifylimiter[key]
+	// first request add timestamp
+	if !found {
+		notifylimiter[key] = time.Now()
+		return true
+	}
+
+	// 10 minutes passed update timestamp
+	if time.Now().After(timestamp.Add(10 * time.Minute)) {
```

Comment:
also lets make it variable so we can control it just like exceptionUrls

Reply:
done!